src/frontend/src/common/types/strings.ts

```diff
@@ -0,0 +1,3 @@
+import { Pathogen } from "src/common/redux/types";
+
+export type StringsType<T> = Record<Required<Pathogen>, T>;
```

Comment:
I don't love `StringsType` as the name: it feels too general, like it could encompass too much. I'm struggling to come up with something better though that isn't a mouthful: `PerPathogenStrings` seems a little awkward. This type is specifically about having a structure for doing lookups of user-facing strings for each kind of pathogen we support. I would like a name that gets that across a bit more.

Reply:
missed this comment previously.  Will address in one of the follow up PRs.